relayer/crates/starknet-chain-components/src/impls/messages/packet.rs

```diff
@@ -221,3 +229,116 @@ where
         Ok(message)
     }
 }
+
+fn from_cosmos_to_cairo_packet<Encoding>(packet: &IbcPacket, encoding: &Encoding) -> CairoPacket
+where
```

Comment:
This conversion feels quite hacky, doesn’t seem worth investing in, especially since we’ll be implementing #150 soon. I’m becoming more convinced to prioritize #150 first (at least for the types used in this PR), then get back here. What do you think?

Reply:
I would like to finish up the handshake and relays, as I already started working on this.

Or, we can merge until here and work on #150 and then resume work on getting packet relay working.

Reply:
I think we have enough time ahead of us, thus not recommend leaving the PR with this function. 
Let's get #150 done first. Let me know if I can be assist on that.

Reply:
Since, I am wrapping up the relay, is it okay if I leave it as it is. The above concern is already mentioned in #150, so I am not open a new issue.